frontend/src/App.js

```diff
@@ -29,6 +29,10 @@ class App extends React.Component {
                 <AppLoader />
                 <Switch>
                     <Route exact path="/main" component={MainView} />
+                    <Route
+                        path="/api/payments/payment-return"
```

Comment:
Any reason you choose a render props instead of a component?

Reply:
fixed.